muddy_macros/src/internal.rs

```diff
@@ -42,7 +42,13 @@ pub(crate) fn build_obfuscation_mod(
                 Some(ref s) => s.as_str(),
                 _ => "MUDDY",
             };
-            eprintln!("{env_name}='{key}'");
+            eprintln!("Please, set {env_name} env variable with content:\n");
```

Comment:
I know that this is complete nitpicking, but the extra `\n` is redundant with `eprintln!`. I also thought that perhaps the same idea you suggested for the `panic` statements might be useful here:

The reason I'm fighting with this line a bit is that it will print out every single build for anybody that is using the library, and we don't have a good way of reducing the noise for those who already know how to use it and don't want to see it every build. (Since this is basically only useful for the user on the first time they test the library, afterwards they know that the following line is the `ENV` var they need to set.)

Do you still think the line is necessary? If so, do you think that the `#[cfg(debug_assertions)]` here would be enough guidance for the user? (I was thinking that maybe the line is not necessary since the info on how to use it is in the rep/on crates.io...). What are your thoughts?


```suggestion
            #[cfg(debug_assertions)]
            eprintln!("Please set the {env_name} env variable with:");
```

Reply:
> If so, do you think that the `#[cfg(debug_assertions)]` here would be enough guidance for the user?

@orph3usLyre, it's the compile-time message, so it will be useful to hint what's wrong for a developer that build a program with release profile.

Reply:
Removed extra newline (`\n`).

Reply:
Alright, merged!